main_full_par_trafo.g

```diff
@@ -44,6 +44,15 @@ ChiefParallel := function( galoisField,mat,a,b )
             h;
 
     ##Preparation: Init and chopping the matrix mat
+
+		##Not supported yet
```

Comment:
It seems like your editor generates tabs instead of real spaces. Change them to spaces to be in line with the other files please :)